.gitignore

```diff
@@ -4,4 +4,5 @@ project/*
 static/*
 dbt_gen/*
 *.yml
-.dccache
\ No newline at end of file
+.dccache
+.DS_Store
```

Comment:
this always pops up for me in my editor, so I thought it was worth while adding it to the ignore file

Reply:
.DS_store looks like a [mac os metadata file](https://en.wikipedia.org/wiki/.DS_Store), agreed it should be ignored. :D